apm-sniffer/apm-sdk-plugin/elasticsearch-6.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/elasticsearch/v6/define/RestHighLevelClientInstrumentation.java

```diff
@@ -260,6 +260,6 @@ public StaticMethodsInterceptPoint[] getStaticMethodsInterceptPoints() {
 
     @Override
     protected String[] witnessClasses() {
-        return new String[] {Constants.TASK_TRANSPORT_CHANNEL_WITNESS_CLASSES};
+        return new String[] {Constants.CREATE_INDEX_RESPONSE_WITNESS_CLASSES};
```

Comment:
So, **TASK_TRANSPORT_CHANNEL_WITNESS_CLASSES** as witness class is working for other interceptors?

Reply:
There are no other problems for the time being.

Reply:
What do you mean? Why does this matter and others are not?

I can see `org.elasticsearch.transport.TaskTransportChannel` is used for 6 instrumentation definitions, if this witness class is changed, but others are not, I would assume, there are still other interceptors are going to run?

Reply:
@xzyJavaX We need feedback about this before we merge this change.

Reply:
We confirm that TASK_TRANSPORT_CHANNEL_WITNESS_CLASSES is working for other interceptors.And the version of transport-client can be changed from 6.7.1-6.8.4 to 6.2.0-6.8.4.Test cases will be added later.